package.json

```diff
@@ -15,7 +15,7 @@
     "install-cli": "cd ./lighthouse-cli && npm install",
     "install-extension": "cd ./lighthouse-extension && npm install",
     "install-viewer": "cd ./lighthouse-viewer && npm install",
-    "build-all": "npm run build-cli && npm run build-extension && npm run build-viewer",
+    "build-all": "npm run build-cli && npm run build-extension && npm run build-viewer && gulp",
```

Comment:
Shouldn't `gulp` be first so the templates are compiled by the time other places need them?